packages/appstore/src/components/add-more-wallets/__test__/wallet-add-card.spec.tsx

```diff
@@ -0,0 +1,132 @@
+                            { account_category: 'wallet', landing_company_name: 'svg', is_virtual: 0, currency: 'USD' },
+                        ],
+                        landing_company_name: 'svg',
+                    },
+                },
+            };
+        }
+
+        if (name === 'get_account_types') {
+            return {
+                data: {
+                    get_account_types: {
+                        wallet: {
+                            crypto: {
+                                currencies: ['BTC'],
+                            },
+                            doughflow: {
+                                currencies: ['USD'],
+                            },
+                        },
+                    },
+                },
+            };
+        }
+
+        if (name === 'website_status') {
+            return {
+                data: {
+                    website_status: {
+                        currencies_config: {
+                            USD: { type: 'fiat', name: 'US Dollar' },
+                            BTC: { type: 'crypto', name: 'Bitcoin' },
+                            UST: { type: 'crypto', name: 'USDT' },
+                        },
+                    },
+                },
+            };
+        }
+
+        return { data: undefined };
+    }),
+}));
+
+describe('AddWalletCard', () => {
+    it('should render currency card', () => {
+        const mock = mockStore({});
+
+        render(
+            <StoreProvider store={mock}>
+                <APIProvider>
+                    <AddWalletCard
+                        wallet_info={{
+                            currency: 'BTC',
+                            gradient_card_class: '',
+                            is_added: false,
+                            landing_company_name: 'svg',
+                        }}
+                    />
+                </APIProvider>
+            </StoreProvider>
+        );
+
+        const add_btn = screen.getByRole('button', { name: /Add/i });
+        expect(screen.getByText('BTC Wallet')).toBeInTheDocument();
+        expect(screen.getByText('SVG')).toBeInTheDocument();
+        expect(add_btn).toBeInTheDocument();
+        expect(add_btn).toBeEnabled();
+        expect(
+            screen.getByText(
+                "Deposit and withdraw Bitcoin, the world's most popular cryptocurrency, hosted on the Bitcoin blockchain."
+            )
+        ).toBeInTheDocument();
+    });
+
+    it('should disabled button when it is disabled', () => {
+        const mock = mockStore({});
+
+        render(
+            <StoreProvider store={mock}>
+                <APIProvider>
+                    <AddWalletCard
+                        wallet_info={{
+                            currency: 'BTC',
+                            gradient_card_class: '',
+                            is_added: true,
+                            landing_company_name: 'svg',
+                        }}
```

Comment:
could we place these values into const variable and just pass it since we're changing only the is_added, something like this:

```suggestion
const wallet_obj = {
  currency: 'BTC',
  gradient_card_class: '',
  is_added: false,
  landing_company_name: 'svg',
};

wallet_info={{ ...wallet_obj, is_added: true }}
````